charts/drax/values.yaml

```diff
@@ -386,6 +390,36 @@ pm-counters:
     #      - chart-example.local
 
 
+notify:
+  enabled: true
+
+  bootstrap:
+    name: "{{ $.Release.Name }}-bootstrap"
+
+  nameOverride: ""
+  fullnameOverride: ""
+
+  containerName: notify
+
+  kafkaInitImage:
+    repository: accelleran/acc-generic-img
+    pullPolicy: IfNotPresent
+    tag: "0.8.0"
+
+  image:
+    repository: python
+    pullPolicy: IfNotPresent
+    tag: "3.12.5"
```

Comment:
I guess we could use python:3.12.5-slim here as a default. We can easily test this once deployed E2E.